src/mtool/api/rest/rest_api/dagent/ibofos.py

```diff
@@ -93,7 +93,7 @@ def send_command_to_dagent(req_type, url, headers, timeout=None, data=None):
                 return response
             if response.status_code == 202 or response.status_code == 503:
                 retry_count = 5
-            if(retry_count >= 5):
+            if(retry_count >= 1): # Removing retry logic as D-Agent has locks implemented
```

Comment:
Not removing the loop, retry_count and time.sleep now, as we are removing the retry since D-Agent has the lock in place. In the coming release, if Poseidon OS allow multiple commands at a time, we need to bring back the retry logic.